implementations/rust/ockam/ockam_command/src/project/info.rs

```diff
@@ -47,6 +48,28 @@ pub struct ProjectInfo<'a> {
     pub okta_config: Option<OktaConfig<'a>>,
 }
 
+impl TryFrom<ProjectLookup> for ProjectInfo<'_> {
+    type Error = ockam::Error;
+    fn try_from(p: ProjectLookup) -> Result<Self, Self::Error> {
+        Ok(Self {
+            id: p.id.into(),
+            name: p.name.into(),
+            identity: p.identity_id,
+            access_route: p
+                .node_route
+                .expect("Project access route is missing")
```

Comment:
For my own education, can this panic instead of returning the error type? Or is that a different `expect` call than the familiar one? I think it's the only fallible piece I see in here.